orb.yml

```diff
@@ -14,24 +14,25 @@ commands:
       internal buildevents orb command.  don't use this.
     steps:
       - save_cache:
-          key: buildevents-v0.4.11
+          key: buildevents-v0.5.1
           paths:
             - /tmp/be
   restore_be_cache:
     description: |
       internal buildevents orb command.  don't use this.
     steps:
       - restore_cache:
-          key: buildevents-v0.4.11
+          key: buildevents-v0.5.1
   download_be_executables:
     description: |
       internal buildevents orb command.  don't use this.
     steps:
       - run:
           name: downloading buildevents executables
           command: |
-            curl -q -L -o /tmp/be/bin-linux/buildevents https://github.com/honeycombio/buildevents/releases/download/v0.4.11/buildevents-linux-amd64
-            curl -q -L -o /tmp/be/bin-darwin/buildevents https://github.com/honeycombio/buildevents/releases/download/v0.4.11/buildevents-darwin-amd64
+            curl -q -L -o /tmp/be/bin-linux/buildevents https://github.com/honeycombio/buildevents/releases/download/v0.5.1/buildevents-linux-amd64
```

Comment:
does it make sense to unconditionally download all 3 at the start even if we're throwing away 2 of them?

Reply:
I think it does - because you could have a workflow with multiple machine-types.

Reply:
oh, I see, it downloads once at the start to the shared workspace?

Reply:
That's my understanding, yeah.